.editorconfig

```diff
@@ -0,0 +1,23 @@
+; top-most EditorConfig file
+root = true
```

Comment:
Not sure if they are going to accept this file as it depends on editor extensions